packages/infrastructure/outputs.tf

```diff
@@ -0,0 +1,29 @@
+output "resource_group_name" {
+  value = azurerm_resource_group.this.name
+}
+
+output "kubernetes_cluster_name" {
+  value = azurerm_kubernetes_cluster.this.name
+}
+
+output "sql_server_name" {
+  value = azurerm_sql_server.this.name
+}
+
+output "sql_database_name" {
+  value = azurerm_sql_database.this.name
+}
+
+output "slack_db_name" {
+  value = azurerm_cosmosdb_account.slack_db.name
+}
+
+output "slack_db_connection_strings" {
```

Comment:
Does `output` stuff need to go in a file called `outputs.tf` or could the slack-related stuff all go together inn a file called `slackbot.tf`?

Reply:
It can be in any .tf file in the folder. But terraform best practises puts it in the outputs.tf file